nailgun/entities.py

```diff
@@ -1124,6 +1128,23 @@ def promote(self, synchronous=True, **kwargs):
         response = client.post(self.path('promote'), **kwargs)
         return _handle_response(response, self._server_config, synchronous)
 
+    def inc_update(self, synchronous=True, **kwargs):
```

Comment:
Can you rename this to `incremental_update`? I'm trying to keep method names identical to the underlying path called. I'm going to add a decorator at some point that'll formalize this convention.